### ISSUE TITLE
Отбор накладываемый на временный таблицы

### ISSUE THREAD

**Issue by Evg-lylyk** [infostart-hub/toolkit#238]
**Существует проблема**
Когда отлаживаешь большой документ проблема обычно в одном документе

**Хотелось бы иметь следующую функциональность**
Установить отбор на ВТ без изменения текста запроса

**Comment by Evg-lylyk** [comment]
Связана с #237

**Comment by Slavdew** [comment]
@Evg-lylyk данный функционал был внедрен? Если да, есть ли описание, если нет, планируется ли внедрение?

**Comment by Evg-lylyk** [comment]
Нет, но планируется. Точно сроки назвать затрудняюсь. Полезная была бы функциональность
